docs/Configuration.md

```diff
@@ -1,26 +1,32 @@
 # Configuration
 
-Default configuration file is located in `src/environments/` folder. All configuration options should be listed in the default configuration file `src/environments/environment.common.ts`. Please do not change this file directly! To change the default configuration values, create local files that override the parameters you need to change. You can use `environment.template.ts` as a starting point.
+Default configuration file is located in `config/` folder. All configuration options should be listed in the default configuration file `src/config/default-app-config.ts`. Please do not change this file directly! To change the default configuration values, create local files that override the parameters you need to change. You can use `appConfig.json` as a starting point.
```

Comment:
Picky language issue:  to *change* the defaults (that is:  to make the *default* values different) editing default-app-config.ts is exactly what a developer should do.  Most people will want to *replace* defaults with local values, and for that one should create local files to override the defaults.  May I suggest "to replace the default..." or "to override the default...."

Reply:
Yeah, this needs a bit of cleanup. Will update and try to resolve the potential confusion.

Reply:
@mwoodiupui I have made an attempt to make this more accurate.